src/collection/dimensions.js

```diff
@@ -266,6 +266,20 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, paddingObject ) {
```

Comment:
Because this only depends only on the width, we may want another style property (enum) to control the values used.  For example, we may want to use

- height
- average of height and width
- min of height and width
- max of height and width

@d2fong What cases would be needed for SBGN usecases?  Do you think it would be best to just support all above cases with another style property?

I.e. `padding-relative-to: width | height | min | max | average`